test/atp/driver/docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
       --workers 1
       --experimental
       --cache-max-pending-records 1
+      --source-timestamp-frequency 100ms
```

Comment:
I think this needs to be just `--timestamp-frequency`

Reply:
ah yeah it looks like thats why testdrive is failing

```
materialized_1     | error: Found argument '--source-timestamp-frequency' which wasn't expected, or isn't valid in this context
materialized_1     | 	Did you mean [32m--[0m[32mtimestamp-frequency[0m?
materialized_1     | 
materialized_1     | USAGE:
materialized_1     |     materialized [OPTION]...
materialized_1     | 
materialized_1     | For more information try --help
```

Reply:
Fixed, thanks.